### ISSUE TITLE
Different hyperparameters between paper and code

### ISSUE THREAD

**Issue by liming-ai** [Megvii-Nanjing/BBN#26]
For iNaturalist:
learning rate is decay at 60th and 80th epoch by 0.1 (in paper), but lr is decay at 120 and 160 in your [code](https://github.com/Megvii-Nanjing/BBN/blob/7992e908842f5934f0d1ee3f430d796621e81975/configs/iNaturalist2018.yaml#L47)

could you please check it?

**Comment by Q-H-Zhang** [comment]
I have the same question. @ZhouBoyan
